src/Markdig.Wpf.SampleApp/MainWindow.xaml.cs

```diff
@@ -13,15 +15,25 @@ public partial class MainWindow : Window
         public MainWindow()
         {
             InitializeComponent();
+            Loaded += OnLoaded;
 
-            var document = Markdig.Markdown.Parse(Test);
+            var markdown = File.ReadAllText("Documents/Markdig-readme.md");
```

Comment:
Yes, good catch - I actually removed this in a later PR, but didn't have time to go back and remove it from here. Will do that.